src/Domain/Model/Daruma.cpp

```diff
@@ -407,24 +418,23 @@ std::string Daruma::getFramesInfo() const
   return info.dump();
 }
 
-int Daruma::getLaunchFrameIndex() const
+std::string Daruma::getLaunchFrameId() const
 {
   if (m_settingsDoc.is_object())
   {
-    return m_settingsDoc[K_LAUNCH_FRAME_INDEX];
+    return m_settingsDoc[K_LAUNCH_FRAME_ID];
```

Comment:
This will throw an exception if no such a field.

Reply:
fixed